packages/vhd-lib/src/Vhd/VhdFile.js

```diff
@@ -458,4 +455,27 @@ export default class Vhd {
       header.parentLocatorEntry[i].platformDataOffset = 0
     }
   }
+
+  readParentLocatorData(parentLocatorId) {
+    assert(parentLocatorId >= 0)
+    assert(parentLocatorId < 8)
+    assert.notStrictEqual(this.header, undefined)
+    const { platformDataOffset, platformDataSpace } = this.header.parentLocatorEntry[parentLocatorId]
+    if (platformDataSpace === 0) {
+      return this._read(platformDataOffset, platformDataSpace)
+    }
+  }
+
+  writeParentLocator(parentLocatorId, data) {
+    assert(parentLocatorId >= 0)
+    assert(parentLocatorId < 8)
+    assert.notStrictEqual(this.header, undefined)
+
+    const { platformDataOffset, platformDataSpace } = this.header.parentLocatorEntry[parentLocatorId]
+    if (platformDataSpace !== 0) {
+      assert(data.length <= platformDataSpace)
+      return this._write(platformDataOffset, data)
+    }
+    assert(data.length === 0)
```

Comment:
Use comparison assert (like `strictEqual` or `notStrictEqual`).

Also, you can simplify a bit the assert by always using strict mode:

```js
import { strict as assert } from 'assert'

assert.equal(foo, 'foo')
```

Reply:
Use the import shown here, it will simplify the code and ensure all equality tests are strict.